org_fedora_oscap/common.py

```diff
@@ -617,3 +625,48 @@ def get_postinst_tailoring_path(data):
         TARGET_CONTENT_DIR,
         data.tailoring_path
     )
+
+
+def get_payload_proxy():
+    """Get the DBus proxy of the active payload.
+
+    :return: a DBus proxy
+    """
+    payloads_proxy = PAYLOADS.get_proxy()
+    object_path = payloads_proxy.ActivePayload
+
+    if not object_path:
+        raise ValueError("Active payload is not set.")
+
+    return PAYLOADS.get_proxy(object_path)
+
+
+def get_packages_data() -> PackagesConfigurationData:
+    """Get the DBus data with the packages configuration.
+
+    :return: a packages configuration
+    """
+    payload_proxy = get_payload_proxy()
+
+    if payload_proxy.Type != PAYLOAD_TYPE_DNF:
+        return PackagesConfigurationData()
```

Comment:
Just a note, this class should be replaced with `PackagesSelectionData` on Fedora Rawhide. The current version will work only on Fedora 34 and RHEL 9.

Reply:
Thanks, time for a RHEL9 branch!

Reply:
see #141